src/Data/ProvisionerInstance.php

```diff
@@ -85,14 +85,14 @@ public function map(Mapper $map) : Provisioner
     }
 
     /**
-     * GetAllRecords must be implemented by concrete implementations of this
+     * GetAllInstances must be implemented by concrete implementations of this
      * class. It must return all rows known about for this Provisioner (without
      * respect to the User accessing the data), which then gets mapped and
      * filtered by execute.
      *
      * @return Traversable of all resources provided by this data source.
      */
-    abstract protected function getAllRecords() : \Traversable;
+    abstract protected function getAllInstances() : \Traversable;
```

Comment:
This was still called "getAllInstances" in the DBRowProvisioner. It had to be either changed there or here, and I changed it here because the word "Instance" is more in line with the other class names..

Reply:
@driusan I am working on it, try to figure out why test cases failed.